src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (Animal.alive === undefined) {
```

Comment:
You don't need to check if Animal.alive is undefined. You can initialize it as a static property directly.

`static alive = [];` and add it before `constructor()`

Reply:
Linter on GitHub [does not allow static fields](https://github.com/mate-academy/js_herbivores_and_carnivores/actions/runs/5256564884/jobs/9498072968?pr=1852#step:5:15). I tried also use `Animal.alive` as property and this throws Linter error locally.